.github/workflows/Rest.yml

```diff
@@ -47,12 +51,16 @@ jobs:
           make release
 
       - name: Start Rest Catalog
-        working-directory: scripts/
         run: |
-          ./start-rest-catalog.sh
+          ./scripts/start-rest-catalog.sh
```

Comment:
Can we maybe add a makefile target for starting the rest catalog too?

I think it's nice if a dev only needs to look at the makefile to understand what to run

Reply:
While doing this, I think its nice to make clear that the requirements to run `make data` are installed in the start-rest-catalog.sh script. Perhaps we can also install the dependencies in a makefile target separately